db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150717110438) do
+ActiveRecord::Schema.define(version: 20150817111030) do
```

Comment:
Hmmm so this file doesn't include a line like:

```ruby
add_index "organisations", ["slug"], name: "unique_organisations_on_slug", unique: true
```

Which means if we load this schema on dev (such as with `rake db:setup`) we won't have the unique constraint anymore. Does this matter?